src/utils/index.ts

```diff
@@ -47,11 +50,64 @@ export const param2Obj = (url: string) => {
   }
   return JSON.parse(
     '{"' +
-    decodeURIComponent(search)
-      .replace(/"/g, '\\"')
-      .replace(/&/g, '","')
-      .replace(/=/g, '":"')
-      .replace(/\+/g, ' ') +
-    '"}'
+      decodeURIComponent(search)
+        .replace(/"/g, '\\"')
+        .replace(/&/g, '","')
+        .replace(/=/g, '":"')
+        .replace(/\+/g, ' ') +
+      '"}'
   )
 }
+interface IDebounce {
+  [key: string]: Function | number | boolean
+}
+
+/**
```

Comment:
Remove this jsdoc, typescript already covered this.

Reply:
Instead, better add a comment to explain what this function used for, see [debounce from Underscore.js](https://davidwalsh.name/function-debounce), also I don't know if the shorter one from underscore.js can do the same thing as this one, I can try it later and replace it if possible.